SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -342,7 +348,14 @@ void USpatialNetDriver::CreateServerSpatialOSNetConnection()
 	check(NetConnection);
 
 	ISocketSubsystem* SocketSubsystem = GetSocketSubsystem();
+	// This is just a fake address so that Unreal doesn't ensure-crash on disconnecting from SpatialOS
```

Comment:
Only in 4.23?

Reply:
The crash only happens in 4.23, but the change should not affect the correctness of 4.22, which is why I didn't wrap it in a guard.